containers/images/core/container-assets/build/10_rake_tasks

```diff
@@ -0,0 +1,40 @@
+#! /bin/bash -e
+
+# Enabling collections
+. /opt/rh/rh-ruby24/enable
+. /opt/rh/rh-nodejs8/enable
+
+export TASKS="db:migrate apipie:cache:index assets:precompile"
```

Comment:
Why `db:migrate` during build?

Reply:
Ah, is this about the dynflow comment below?

Reply:
`apipie:cache:index` needs a database. The comment below about dynflow is only partially correct because AFAIK we already needed the DB before dynflow was in core.

Reply:
i assume we don't need this, but this is a workaround as dynflow expects the db to be there, I'll test it and remove if it works without it.

Reply:
Not just dynflow, apipie:cache, precompile assets (although maybe I coded around that one) both do as well. Basically, ANY rake task needs a populated database because it loads the Rails environment.